.vsts-ci/templatesGeneric/deserializeTags.ps1

```diff
@@ -0,0 +1,26 @@
+$json = Get-ChildItem "${ENV:PIPELINE_WORKSPACE}/releaseTags.json" -recurse -File
+if ($json.Count -ge 1) {
+    $jsonText = Get-Content -Path $json.FullName
+    $releaseTags = $jsonText | ConvertFrom-Json
+    Write-Verbose 'got json' -verbose
```

Comment:
Should this write the `$jsonText`?

Reply:
These are more markers for how far the code has run.  It was failing when I set it up as inline code in the task.  Determined there is some bug in the AzDevOps PowerShell task because my code was never running

Since they have separate logs, I don't think I need to remove them.